pandas/core/generic.py

```diff
@@ -12468,7 +12468,7 @@ def first_valid_index(self) -> Hashable | None:
 
         Notes
         -----
-        If all elements are non-NA/null, returns None.
```

Comment:
Could you just remove this `Notes` section and demonstrate these points in the `Examples` section below?

Reply:
Done. One thing I was unsure of was how to handle returning `None` in the docs:
- I ran it through the docstring validation script `python scripts/validate_docstrings.py pandas.core.generic.NDFrame.first_valid_index`
- It was throwing errors about the output not matching the expected output because I had originally specified:
```
        >>> s = pd.Series()
        >>> s.first_valid_index()
        None
        >>> s.last_valid_index()
        None
```

I considered printing the outputs so that the outputs would match but didn't see print statements used in that way in other parts of the code so I did not keep them. Let me know if printing is the way to go here and I can make the change :)

Reply:
To clarify, option 1:
```
>>> s = pd.Series()
>>> s.first_valid_index()
>>> s.last_valid_index()
```

Option 2:
```
>>> s = pd.Series()
>>> print(s.first_valid_index())
None
>>> print(s.last_valid_index())
None
```

Reply:
Option 2 would be fine to make it explicit in the docs

Reply:
Added the print statements